src/locales/de_DE/de_DE.json

```diff
@@ -1,4 +1,98 @@
 {
+    "auth_intro": {
+        "title": "Welcome back!",
+        "subtitle": "To make sure it’s you, please scan your face",
+        "line1": "You’ll be asked to keep your face within an oval frame.",
+        "line2": "Make sure your eyes are clearly visible.",
+        "button_primary": "Start camera"
+    },
```

Comment:
This object was not made available by lokalise :/ will ping rae for this.

Reply:
@raemarielee FYI ^

Reply:
@kopijunkie I've spoken with rae regarding these- overall they will **not** be added to the SDK itself, as the `AuthIntro` screen is meant to be implemented by the integrator themselves (or, in our our case, most likely a screen on Client Trial App). I've since removed the `AuthIntro` screen from the `Auth` flow on StepComponent, and will proceed to erase these entries to keep it tidy.